fbw-common/src/systems/shared/src/publishers/Msfs/MsfsFlightModelPublisher.ts

```diff
@@ -10,9 +10,11 @@ import {
 interface MsfsFlightModelBaseEvents {
   /** Gets the openess in percent of an interactive point, usually doors */
   msfs_interactive_point_open: number;
+  /** Gets target in percent of an interactive point, usually doors */
```

Comment:
It would be better to copy the actual docs from https://docs.flightsimulator.com/html/Programming_Tools/SimVars/Aircraft_SimVars/Aircraft_FlightModel_Variables.htm#INTERACTIVE_POINT_GOAL, rather than half of it (ideally also for the one above). The idea is that these publishers are just direct MSFS SDK publishers that exactly match the SDK docs unless there's a clear error with them.